.github/workflows/check-go-versions.yml

```diff
@@ -27,26 +27,17 @@ jobs:
         run: |
           echo "latest=${{ fromJSON(env.fetch-api-data)[0].cycle }}" >> $GITHUB_OUTPUT
           echo "penultimate=${{ fromJSON(env.fetch-api-data)[1].cycle }}" >> $GITHUB_OUTPUT
-
-
   create-prs:
     permissions:
       contents: write
       pull-requests: write
     needs: check-go-eol
     runs-on: ubuntu-latest
-    strategy:
```

Comment:
Missed this before, right now the version checker runs on (nonexistent) branches.